neqo-transport/src/stream_id.rs

```diff
@@ -107,6 +87,17 @@ impl StreamId {
     pub fn is_recv_only(self, my_role: Role) -> bool {
         self.is_uni() && self.is_remote_initiated(my_role)
     }
+
+    pub fn next(&mut self) {
+        self.0 += 4;
+    }
+
+    pub fn role_bit(role: Role) -> u64 {
```

Comment:
Please comment that this is the bit shared by all streams created by endpoints in the identified role.